app/containers/Moderation/Content.jsx

```diff
@@ -0,0 +1,239 @@
+/* eslint react/jsx-no-bind: 0, jsx-a11y/click-events-have-key-events: 0, jsx-a11y/no-noninteractive-element-interactions: 0 */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+import messages from 'common-messages';
+
+import createdHistory from 'createdHistory';
+
+import textBlockStyles from 'text-block-styles';
+
+import {
+  BORDER_SECONDARY,
+  BG_SECONDARY_SPECIAL_4,
+  BG_TRANSPARENT,
+  BORDER_TRANSPARENT,
+  TEXT_PRIMARY,
+  TEXT_DARK,
+  BORDER_PRIMARY_LIGHT,
+} from 'style-constants';
+
+import plusIcon from 'images/Plus.svg?inline';
+import minusIcon from 'images/Minus.svg?inline';
+import okayGreen from 'images/okayGreen.svg?external';
+import notOkay from 'images/notOkayRed.svg?external';
+
+import H4 from 'components/H4';
+import Span from 'components/Span';
+import { IconSm } from 'components/Icon/IconWithSizes';
+import BaseRoundedNoPadding from 'components/Base/BaseRoundedNoPadding';
+import BaseTransparent from 'components/Base/BaseTransparent';
+import Button from 'components/Button/Outlined/PrimaryLarge';
+
+export const TextBlock = styled.div`
+  display: ${x => (x.isOpened ? 'block' : 'none')};
+  margin-top: ${x => (x.isOpened ? '15px' : '0px')};
+
+  ${textBlockStyles};
+
+  > * {
+    margin-bottom: 5px;
+  }
+`;
+
+const SectionStyled = BaseRoundedNoPadding.extend`
+  margin-bottom: 15px;
+
+  h4,
+  h5 {
+    cursor: pointer;
+  }
+
+  > :not(:last-child) {
+    border-bottom: ${x => (x.isOpened ? '1' : '0')}px solid ${BORDER_SECONDARY};
+  }
+
+  ${Button} {
+    margin-left: 43px;
+    margin-bottom: 10px;
+  }
+`;
+
+const ImgWrapper = styled.div`
+  margin-right: 18px;
+  width: 42px;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+
+  :hover {
+    cursor: pointer;
+  }
+
+  @media only screen and (max-width: 576px) {
+    margin-right: 8px;
+  }
+`;
+
+const PermissionBox = BaseTransparent.extend`
+  display: flex;
+  align-items: baseline;
+  padding: 10px 30px;
+  background: ${x => (x.isOpened ? BG_SECONDARY_SPECIAL_4 : BG_TRANSPARENT)};
+  border: 1px solid
+    ${x => (x.isOpened ? BORDER_PRIMARY_LIGHT : BORDER_TRANSPARENT)};
+
+  h5 span {
+    color: ${x => (x.isOpened ? TEXT_PRIMARY : TEXT_DARK)};
+  }
+
+  &:first-child {
+    padding-top: 15px;
+  }
+
+  &:last-child {
+    padding-bottom: 15px;
+  }
+`.withComponent('li');
+
+const PermissionBoxBody = styled.div`
+  width: 100%;
+`;
+
+const Permission = ({
+  description,
+  title,
+  permissionCode,
+  sectionCode,
+  getPermissionCode,
+  permission,
+}) => {
+  const questionId = getPermissionCode(sectionCode, permissionCode);
+  let ico;
+  if (permission.includes(permissionCode)) {
```

Comment:
curly braces are inconsistent. If use them in if statement then should use them for else as well.

Reply:
That assignment can be simplified with the conditional:
`const ico = permission.includes(permissionCode) ? okayGreen  : notOkay ;`